pkg/http-handler/http-handler_test.go

```diff
@@ -260,6 +262,27 @@ func TestGetNamespaces(t *testing.T) {
 			err:          k8sErrors.NewForbidden(schema.GroupResource{}, "foo", fmt.Errorf("nope")),
 			expectedCode: 403,
 		},
+		{
+			name:         "it should return the list of namespaces from header and a 200 if the repo is created",
+			namespaces:   []corev1.Namespace{{ObjectMeta: metav1.ObjectMeta{Name: "ns1"}}, {ObjectMeta: metav1.ObjectMeta{Name: "ns2"}}},
+			expectedCode: 200,
+			headerName:   "X-Consumer-Groups",
+			headerValue:  "namespace:ns1, namespace:ns2",
+		},
+		{
+			name:         "it should return the list of namespaces and a 200 when header does not match KUBEAPPS_NAMESPACE_HEADER",
+			namespaces:   []corev1.Namespace{{ObjectMeta: metav1.ObjectMeta{Name: "foo"}}},
+			expectedCode: 200,
+			headerName:   "X-Consumer-Groups",
+			headerValue:  "nspaces:ns1, nspaces:ns2",
+		},
+		{
+			name:         "it should return the list of namespaces and a 200 when when header does not match KUBEAPPS_NAMESPACE_PATTERN",
```

Comment:
You need to update these descriptions right? `KUBEAPPS_NAMESPACE_PATTERN` should instead refer to the Helm chart value?